eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml

```diff
@@ -9,9 +14,9 @@ stages:
       vmImage: MMSUbuntu18.04
     steps:
       - task: UseDotNet@2
-        displayName: 'Use .NET Core sdk $(DotNetCoreVersion)'
+        displayName: 'Use .NET Core sdk ${{ coalesce( parameters.DotNetCoreVersion, variables.DotNetCoreVersion) }}'
```

Comment:
@weshaggard  I believe this is the only piece that affects the greater repo.

Reply:
We should probably clean these up and make everyone use the newer version of .NET but I'm ok with this as a temporary change. cc @chidozieononiwu something we should try and clean-up when we start doing clean-up work in this repo.